cmd/wksctl/profile/disable/disable.go

```diff
@@ -0,0 +1,100 @@
+package disable
+
+import (
+	"errors"
+	"fmt"
+	"os"
+	"path"
+
+	log "github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+	"github.com/weaveworks/wksctl/cmd/wksctl/profile/constants"
+	"github.com/weaveworks/wksctl/pkg/git"
+)
+
+var Cmd = &cobra.Command{
+	Use:   "disable",
+	Short: "disable profile",
+	Args:  profileDisableArgs,
+	Run:   profileDisableRun,
+}
+
+var profileDisableOptions struct {
+	repository string
+	noCommit   bool
+}
+
+func init() {
+	Cmd.Flags().StringVarP(&profileDisableOptions.repository, "repository", "", "", "enable profile from the repository")
```

Comment:
Flags should not be `P` if you don't provide the short version.

Reply:
fixed